### ISSUE TITLE
Send message annotation in a separate msg attribute

### ISSUE THREAD

**Issue by arsu77** [stefanopog/node-red-contrib-wws#3]
Currently the annotation payload of a message-annotation-added event is returned as msg.payload.
It would be better to contain both (payload -> the original request) and annotationPayload (containing the JSON parsed annotation payload)

SHOULD:
msg.payload
msg.annotationPayload

**Comment by stefanopog** [comment]
Yes, and IF POSSIBLE, making the parsing of strings->Json RECURSIVE,
Do not know if it is possible though

**Comment by arsu77** [comment]
Stefano, I think this issue has already been solved by you, correct?
